src/services/util/helpers.ts

```diff
@@ -0,0 +1,19 @@
+interface anyObj{
+    [key: string]: any
+}
+export function createConfigArgsList(obj:anyObj): anyObj[]{
+    console.log(obj)
+    let argsList: anyObj[] = [];
+    for(const key of Object.keys(obj)){
+        console.log(key, obj[key])
+        if(obj[key] != undefined){
+            const arg = {} as anyObj;
+            arg[key] = obj[key];
+            argsList.push(
+               arg
+            )
+        }
+    }
+    console.debug(argsList)
+    return argsList
```

Comment:
can you put a docstring for the function to provide a brief explanation of what it does?

Reply:
0bdfc6179361ff786d079973fbdac83a061079f8 -- also changed how arg and return type is displayed so is more explicit.